Core/Core/Database.xcdatamodeld/Database.xcdatamodel/contents

```diff
@@ -249,6 +249,7 @@
     <entity name="Conversation" representedClassName="Core.Conversation" syncable="YES">
         <attribute name="audienceIDsRaw" attributeType="String"/>
         <attribute name="avatarURL" attributeType="URI"/>
+        <attribute name="cannotReply" optional="YES" attributeType="Boolean" usesScalarValueType="YES"/>
```

Comment:
The property is marked as optional here but in the Conversation entity it's not an optional. I think a non optional field makes more sense so let's update the database model based on the implemented property.